### ISSUE TITLE
DAC ACP <> Average Operations

### ISSUE THREAD

**Issue by shahzadlone** [sourcenetwork/defradb#2475]
Parent Issue #1738 

This issue is to make sure we properly test access control (ACP) in combination to ensure there aren't any bugs, there might be some missing implementation that might need to be done or it might just work.

**Comment by ribiza** [comment]
@shahzadlone @islamaliev Should this have been closed as resolved by https://github.com/sourcenetwork/defradb/pull/2583?

**Comment by shahzadlone** [comment]
> @shahzadlone @islamaliev Should this have been closed as resolved by https://github.com/sourcenetwork/defradb/pull/2583?

Yes i believe so the 'and' keyword between issues perhaps missed it. Good catch.
